examples/server/webui/src/main.js

```diff
@@ -468,14 +468,22 @@ const mainApp = createApp({
         for await (const chunk of chunks) {
           const stop = chunk.stop;
           const addedContent = chunk.choices[0].delta.content;
-          const lastContent = this.pendingMsg.content || '';
+          const lastContent = this.pendingMsg.fullContent || '';
           if (addedContent) {
             this.pendingMsg = {
               id: this.pendingMsg.id,
               role: 'assistant',
-              content: lastContent + addedContent,
+              fullContent: lastContent + addedContent,
             };
           }
+          const regex = /<think>(.*?)?(?=(<\/think>)|$)/gis;
```

Comment:
Personally I'm not a fan of abusing `(.*?)` regex, because it can be bad for performance (and even lead to infinite backtracking in some cases). You should rely on something like `.split()` for be safer